reactivesocket-publishers/src/main/java/io/reactivesocket/reactivestreams/extensions/internal/publishers/TimeoutPublisher.java

```diff
@@ -28,6 +28,15 @@
 
 public final class TimeoutPublisher<T> implements Px<T> {
 
+    @SuppressWarnings("ThrowableInstanceNeverThrown")
+    private static final TimeoutException timeoutException = new TimeoutException();
+
+    private static final StackTraceElement[] EMPTY_STACK = new StackTraceElement[0];
```

Comment:
Makes sense. I will update.
BTW, this code is in publishers package and `TimeoutException` is in core, so this code here uses `java.util.concurrent.TimeoutException`